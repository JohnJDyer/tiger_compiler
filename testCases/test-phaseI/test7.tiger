let
  var X: int := 0;
in
  printi(1 + 2 - 3 * 4 + 5 + 6);
end
