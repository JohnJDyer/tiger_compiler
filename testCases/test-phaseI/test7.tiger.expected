	assign, X, 0, 
main:
	add, 1, 2, $t0
	mult, 3, 4, $t1
	sub, $t0, $t1, $t2
	add, $t2, 5, $t3
	add, $t3, 6, $t4
	call, printi, $t4
	return, , , 
